### PR TITLE
If the cashier tries to add something that has more than the amount i…

### PR DESCRIPTION
**Pull request by calvinbrown085**
…n the db it won't allow it

Closes: https://github.com/ethanphunter/PointOfSaleProject/issues/69

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 app.config.update(dict(
     #DEBUG=True,
-    SECRET_KEY= os.environ["Key"]))
+    SECRET_KEY= """os.environ["Key"]"""))
```
Comment (ethanphunter) [inline main.py]: This needs to get changed back

**Comment by ethanphunter** [comment]
Other than my comment 👍

**Comment by calvinbrown085** [comment]
@ethanphunter fixed

**Comment by ethanphunter** [comment]
:+1:

**Comment by gilmerc** [comment]
:+1:

**Pull request merged**